### ISSUE TITLE
Ensure prefix maps is a true bimap

### ISSUE THREAD

**Issue by matentzn** [mapping-commons/sssom-py#469]
In the latest release (not necessarily on master, have not checked), duplicate URI prefixes may enter. Write a test that ensures that when an external prefixmap is passed in during `parse` that overlaps with the internal prefix map, that not both get serialised:

```
#   Orphanet: http://www.orpha.net/ORDO/Orphanet_
#   orphanet.ordo: http://www.orpha.net/ORDO/Orphanet_
```

- Blocked by https://github.com/cthoyt/curies/issues/96

**Comment by cthoyt** [comment]
Related: https://github.com/biopragmatics/bioregistry/pull/969

**Comment by matentzn** [comment]
No longer relevant!
